README.md

```diff
@@ -39,35 +51,131 @@ class ListMovingVehicle(vqpy.QueryBase):
+
+```python
+class People_loitering_query(vqpy.QueryBase):
+
+    @staticmethod
+    def setting() -> vqpy.VObjConstraint:
+        REGIONS = [[
+            (550, 550), (1162, 400), (1720, 720), (1430, 1072), (600, 1073)
+        ]]
+
+        filter_cons = {
+            "__class__": lambda x: x == Person,
+            "bottom_center": vqpy.utils.continuing(
+                condition=vqpy.utils.within_regions(REGIONS),
+                duration=10, name="in_roi"
+            ),
+        }
+        select_cons = {
+            "track_id": None,
+            "coordinate": lambda x: str(x),  # convert to string for JSON serialization
+            "in_roi_periods": None,
+        }
+        return vqpy.VObjConstraint(filter_cons, select_cons)
+```
+
 #### Launch the task
 
-Last, we can call `vqpy.launch` to start query video frames.
+After declaring `VObj` and `Query`, users should call the `vqpy.launch` function to deploy the video query, below is an example of this interface. Users should specify the detector class they expect, and map the object detection result to the defined VObj types.
 
 ```python
 vqpy.launch(cls_name=vqpy.COCO_CLASSES, # detection class
             cls_type={"car": Vehicle, "truck": Vehicle}, # mappings from detection class to VObj
             tasks=[ListMovingVehicle()], # a list of Queries to apply
             video_path=args.path, # the path of the queried video
-            save_folder=args.save_folder # result of query will be saved as a json file in this folder
+            save_folder=args.save_folder, # result of query will be saved as a json file in this folder
+            detector_name="yolox", # optional, specify the name of user's own model
+            detector_model_dir=args.pretrained_model_dir # optional, specify the directory of user's pretrained model
             )
 ```
 
-Under the hood, VQPy will automatically select an object detection model that outputs the specified `cls_name`. Multiple video optimizations will be conducted transparently to improve the end-to-end video query performance. 
+Under the hood, VQPy will automatically select an object detection model that outputs the specified `cls_name`, and decide the workflow to generate the object properties. Multiple video optimizations will be conducted transparently to improve the end-to-end video query performance.
 
-### Customization
+#### Detector and tracker
 
-For more details on customization, please refer to the document [here](https://github.com/uclasystem/VQPy/blob/main/docs/frontend.md#customization).
+Under normal conditions, users do not need to provide a detector implementation. However, if the user have a detector, they can inherit `vqpy.detector.DetectorBase` and implement the required interfaces.
```

Comment:
Shall we keep it as a reference link to the customization doc?  And made modifications there?
I want to keep the get started doc simple, to give users a general idea about how to use vqpy.

Reply:
I think the current document covers the necessary information in `docs/frontend.md`. Are there anything that is required and cannot be added to the API document?

Reply:
I am not sure I understand... I mean, we can put the customization part to another document, like `docs/frontend.md`.